racetrack_client/README.md

```diff
@@ -1,30 +1,93 @@
-# Racetrack client context
-`racetrack-client` is a CLI client tool for deploying workloads to Racetrack.
+# Racetrack CLI client
+`racetrack-client` is a CLI client tool for deploying and managing workloads in Racetrack.
 
 Racetrack system allows to deploy jobs in a one step.
 It transforms your code to in-operation workloads, e.g. Kubernetes workloads.
 You write some code according to a set of coventions, you include the manifest file which explains the code, 
 and you submit it to Racetrack. A short while after, the service calling your code is in operation.
 
-# Installation
+## Quickstart
+```shell
+# Install racetrack-client
+pip3 install racetrack-client
+
+# Set current remote
+racetrack set remote https://racetrack.platform.example.com
+
+# Log in
+racetrack login T0k3n.g0es.H3r3
+
+# Deploy a Job
+racetrack deploy
+```
+
+## Installation
 Install racetrack-client using pip:
-```bash
+```shell
 pip3 install racetrack-client
 ```
 
-Python 3.8+ is required. So if you have any troubles, try with:
+Python 3.8 (or higher) is required.
+
+This will install `racetrack` CLI tool. Verify installation by running `racetrack` command.
+
+## Usage
+Run `racetrack --help` to see usage.
+
+### Adding a remote
+Assuming your Racetrack server is running on https://racetrack.platform.example.com/lifecycle,
+you can add this remote as an alias:
+```shell
+racetrack set alias my-dev https://racetrack.platform.example.com/lifecycle
```

Comment:
`alias` might also need explaining. We can't assume users will hop around between different docs, so it isn't enough to put it in the glossary.

While it might not be possible to achieve 100%, the more we write a doc as if users haven't read the others the better it is (of course, it's a fine balance to avoid actual duplication).